.github/workflows/ci.yaml

```diff
@@ -106,6 +106,11 @@ jobs:
           CUDA_VER: "11.5.1"
           CENTOS_VER: "8"
 
+        - DOCKERIMAGE: linux-anvil-cuda
+          DOCKERTAG: "11.6"
+          CUDA_VER: "11.6.0"
+          CENTOS_VER: "8"
```

Comment:
Is there a reason we moved to CentOS 8 for CUDA 11.6 / 11.5 on x86? CentOS 7 is still supported as far as I can tell?

Reply:
Interesting I seem to recall CentOS 7 wasn't supported here before and recall us discussing this, but can't find that atm. Asking here ( https://github.com/conda-forge/docker-images/pull/198#discussion_r787116632 )

Reply:
cc @ajschmidt8 @leofang

Reply:
@jakirkham This is a question we should ask the image providers internally. If you search on the docket hub, there's no cos7 based image for CUDA 11.5.1, but there is one for 11.6.0.

Reply:
Please see this issue ( https://gitlab.com/nvidia/container-images/cuda/-/issues/145 )

Reply:
So should we use CentOS 7 for CUDA 11.6?

Reply:
Checked with others and CentOS 7 should be supported by both. We just lack an image for 11.5 atm.

Yeah switching this to CentOS 7 makes sense. Would also suggest switching the 11.5 images to CentOS 7 once available. Have raised issue ( https://github.com/conda-forge/docker-images/issues/205 ) on this point. Am also subscribed to the upstream issue.

Reply:
```suggestion
          CENTOS_VER: "7"
```